user_guide_src/source/incoming/controllers.rst

```diff
@@ -26,13 +26,14 @@ Let's try it: Hello World!
 ==========================
 
 Let's create a simple controller so you can see it in action. Using your text editor, create a file called Helloworld.php,
-and put the following code in it::
+and put the following code in it. You will notice that the Helloworld Controller is extending the BaseController you can
+also extend the Controller if you do not need the functionality of the BaseController.::
 
     <?php namespace App\Controllers;
 
-    use CodeIgniter\Controller;
+    use CodeIgniter\BaseController;
```

Comment:
The `BaseController` is in the `App\Controllers` namespace so there is no need for `use ...` here.

Maybe it would make sense to leave one example that uses `Controller` so users could understand the difference between using `Controller` and `BaseController` when it comes to the "loading" part? I don't want the newbies to feel lost.

Reply:
Ok, I'll layout better for everyone to understand I changed it because the frameworks Home controller is using.

Reply:
Yes, I get it.